src/components/ChooseSubmodule/ChooseSubmodule.tsx

```diff
@@ -0,0 +1,98 @@
+import { ArrowBackIcon, ChevronDownIcon } from "@chakra-ui/icons";
+import { Button, Flex, useDisclosure, useToken } from "@chakra-ui/react";
+import type { Assembly } from "jsii-reflect";
+import { FunctionComponent, useCallback, useMemo, useState } from "react";
+import { useHistory, useLocation } from "react-router-dom";
+import { useQueryParams } from "../../hooks/useQueryParams";
+import { SearchPopover } from "./SearchPopover";
+
+export interface ChooseSubmoduleProps {
+  assembly?: Assembly;
+}
+
+export const ChooseSubmodule: FunctionComponent<ChooseSubmoduleProps> = ({
```

Comment:
If we want to move towards `const x: FunctionComponent<props> = () => {}` we should do it for everything. Keeping consistent style would be my preference for now.

Reply:
Fair enough, figured I'd just go ahead & start moving towards that direction, but I can hold off

Reply:
lets do it all at once, and add a lint rule to prefer this going forward. do it now! but just separately.